### DISCUSSION TITLE
Zebra printer compatibility

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3346] [Q&A]
Hello Pretix team!

Is the Zebra ZD230 printer compatible with pretix to print tickets in box office ?

Thank you in advance

**Comment by raphaelm** [comment]
If you print using a PC, any printer is compatible with the right driver.

If you print using our pretixPRINT Android app, yes, the ZD230 *should* be compatible, but we've never tested it.

Here are the printers we officially recommend and support: https://pretix.eu/about/en/hardware/pos
